### ISSUE TITLE
Can't change equation font.

### ISSUE THREAD

**Issue by smnajmul19** [ONLYOFFICE/DesktopEditors#1705]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my idea.

### Your idea.

in equation editor I can't change default font. But I am Bangladeshi. I need to write Bengali in equation editor and bengali math. But unfortunately i can't write bengali in equation editor.

**Comment by TatianaKirillova** [comment]
Hello @smnajmul19 . Thank you for contacting us!
What exactly is the problem? Are the symbols adding up incorrectly or something else?

**Comment by smnajmul19** [comment]
![sample](https://github.com/user-attachments/assets/bfd3adf3-b7d1-4c14-b864-47bf7e3c00d1)

I'm unable to change the default font in OnlyOffice, even though I can do it in other office suites like MS Office and LibreOffice. I've tried changing the font in the equation editor but it's not working.

**Comment by TatianaKirillova** [comment]
@smnajmul19 Got your point, could you please attach the font you are using to check the problem?

**Comment by smnajmul19** [comment]
No need to attach any font. I can't change default font cambria to any font. If I select other font, nothing changed. 
As you requested, I've attached the font. If you apply this font, it will look like the second example of image I provided.
[sutonnymj-regular.zip](https://github.com/user-attachments/files/17523570/sutonnymj-regular.zip)

Os-Ubuntu 24.4
Onlyoffice 8.1

**Comment by TatianaKirillova** [comment]
@smnajmul19 Thank you!

**Comment by TatianaKirillova** [comment]
@smnajmul19 It turns out that the formula signs should be in Cambria Math such as + - and others, and the numbers in Bengali should be in the SutonnyMJ font. Did I understand you correctly?

**Comment by smnajmul19** [comment]
Yes. Number and unicode text should change with my suggested font.

**Comment by TatianaKirillova** [comment]
@smnajmul19 I can confirm that the problem exists, issue #71275 for your problem.

The problem is reproduced on Document Server.
